project/Build.scala

```diff
@@ -101,6 +101,13 @@ object MyScalaJSPlugin extends AutoPlugin {
           NodeJSEnv.Config().withSourceMap(wantSourceMaps.value)),
 
       scalaJSLinkerImpl := {
+        /* !!! This custom implementation does *not* respect the contract of
+         * scalaJSLinkerImpl, because it does not cache the `LinkerImpl` in
+         * scalaJSLinkerImplBox. The reason is that we need a new class loader
+         * to be created every time, in case the linker sources have changed.
```

Comment:
This is not really true: We store the linker *instance* anyways so we need a reload if we want a new linker. So IIUC, there is no reason to not cache the linker impl as well.

Reply:
Right, I guess that's true, indeed.